### DISCUSSION TITLE
这个要把无法购买改成可购买吗？

### DISCUSSION THREAD

**Discussion by yingjiaze** [mikumifa/biliTickerBuy#377] [提问专区] [answered]
配置文件是上海·BilibiliWorld 2024-2024-07-13 周六 - 普通票 - ￥128.0 - 无法购买-要改可购买吗

**Comment by yingjiaze** [comment]
"上海·BILIBILI MACRO LINK 2024-2024-07-13 周六 - C档 - ￥780.0- 未开售 - 【起售时间：2024-07-06 12:00:00】现在是这个，需要改吗

**Comment by SouthSeafireworks** [comment, marked as answer]
这个只是未开售目前的状态，不需要改

**Reply by yingjiaze** [reply]
就是这个不影响，可以直接放是吧，谢谢

**Reply by SouthSeafireworks** [reply]
对的，到时候也可以抢
